kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/security/keystore/KeystoreService.java

```diff
@@ -163,6 +199,36 @@ public void createKeyPair(String alias, String algorithm, int keySize, String si
      */
     public String getCSR(String alias, X500Principal principal, String signerAlg) throws KuraException;
 
+    /**
+     * Creates and returns a CSR for the given keypair based on the provided principal and signer algorithm selected
+     *
+     * @param keyPair
+     * @param principal
+     * @param signerAlg
+     * @return
+     * @throws KuraException
```

Comment:
The return parameter is missing

Reply:
Also, missing since annotation

Reply:
Here the method is present: https://github.com/eclipse/kura/blob/0c002eb17beb4921abb3cc1d64eaf1a57a7a6e54/kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/security/keystore/KeystoreService.java#L150

Reply:
True, but I'm referring to the two getCSRAsPKCS10Builder methods that are now, right?